input/pagecontent/reference.md

```diff
@@ -3206,6 +3217,20 @@ Following [standard profiling rules established in FHIR](https://www.hl7.org/fhi
   * action.definition[x] only Canonical(ActivityDefinition or PlanDefinition)
   ```
 
+{%include tu-div.html%}
+* Restrict MedicationRequest.reason, a choice of CodeableReference(Condition \| Observation), to allow only a CodeableReference to an Observation
+
+  ```
+  * reasion only CodeableReference(Observation)
```

Comment:
```suggestion
  * reason only CodeableReference(Observation)
```

Reply:
Oh wow, whoops!

Reply:
Fixed in [f3be4f8](https://github.com/HL7/fhir-shorthand/pull/159/commits/f3be4f84ccbd40ecb37e16b5474e41dbb020681a).